src/components/PhotoMap/PhotoMap.js

```diff
@@ -40,7 +41,7 @@ class PhotoMap extends Component {
   render() {
```

Comment:
Sounds good! I would only be able to get to this tonight, though, so if you want to work on the project in the meantime with this code feel free to squash and merge. I promise to make the change in the next 'round.